app/webpacker/styles/vertical-tags.scss

```diff
@@ -19,8 +19,8 @@
     display: inline-block;
     margin-left: -2em;
 
-    color: $white;
-    border: 1px solid $purple;
-    background: $purple;
+    color: $black;
+    border: 1px solid $blue;
+    background: $blue;
```

Comment:
Should we save blue for the h2s @garethwilliamsdesign - is it a bit confusing to use blue for flag content?

Reply:
I know that the headings on these pages are other colours though, so maybe it doesn't matter!